### DISCUSSION TITLE
Repo de Katas en GitHub

### DISCUSSION THREAD

**Discussion by carezus** [LaunchX-InnovaccionVirtual/CursoIntroPython#166] [Q&A]
Si las Katas las hice aparte en notebooks y las subí al repo arrastrándolos de la carpeta, es válido?

**Comment by OscarPedroza** [comment]
Si es válido, siempre y cuando entregues el link del repositorio con el formato solicitado.

**Comment by richirrim** [comment]
Si, justamente eso fue lo que menciono en el live que acaba de terminar, mientras entregues tus katas todo es valido.

**Comment by DanielChavezZ** [comment]
si siempre y cuando sigas las indicaciones proporcionadas en el live

**Comment by JavGom201** [comment]
¿El nombre del Repositorio para entregar las katas tiene algún formato en específico?

**Reply by DanielOe21** [reply]
No, el nombre que tú gustes ponerle al repositorio está bien.

Verifica que el link debe ser con el formato de GitHub/usuario/repositorio

**Reply by JavGom201** [reply]
Excelente, muchas gracias!
